farmdata2_modules/fd2_tabs/resources/CustomTableComponent.js

```diff
@@ -364,10 +364,10 @@ let CustomTableComponent = {
 
             for(let i = 0; i < this.rows.length; i++){
                 for(let j = 0; j < this.columns.length; j++){
-                    if(this.columns[j].visible){
+                    if(this.columns[j].visible && this.indexesToAction.includes(this.rows[i].id)){
                         if(typeof this.rows[i].data[j] === 'string'){
                             cleanHTML = this.rows[i].data[j].replaceAll(/(<p[^>]+?>|<p>|<\/p>|<br \/>)/img, "")
-                            cleanHTML = cleanHTML.replaceAll(/(\r\n|\n|\r)/gm, "-")

```

Comment:
I seem to recall that there was a reason we were using `-` and `--` here.  I think it had to do with inserting new lines or columns not aligning properly in the CSV.  Double check the output that is generated to ensure that just using here doesn't mash stuff together and make it unreadable.  Also be sure to understand when `-` and `--` are used so that we address all instances of issues, even if they don't show up in the simple example produced by the steps in the issue.

Reply:
This seems to be guaranteed. I didn't see any problems when I made my changes on my end.

Reply:
I think we need to revert this change and continue replacing all new line characters with a `-`.  Otherwise what is happening is that any comments that contain new lines to separate items or phrases are being mashed together with no space between the items.  You can observe this in the second comment in the seeding report generated using the default dates.  In the rendered table in FarmData2 the comment is broken into multiple lines.  In the exported CSV the lines are mashed together with no spaces.  We had added the `-` just to keep the items visually separated, even though they will be on a single line in the CSV.  We might consider using " - " a dash with a space on either side instead to make it even a little better.

Reply:
This has been made the case in the latest commits.